airbyte-webapp/src/hooks/services/Notification/NotificationService.tsx

```diff
@@ -35,6 +35,9 @@ export const NotificationService = React.memo(({ children }: { children: React.R
         <Toast
```

Comment:
Not related to your changes, but the `.actionButton` style in `Toast` has a margin top that makes it unaligned: 
<img width="525" alt="image" src="https://user-images.githubusercontent.com/7550957/217344350-2c43f0fe-dd4c-4429-9a8c-97f92b66be2f.png">

Can we remove that?

After:

<img width="521" alt="image" src="https://user-images.githubusercontent.com/7550957/217344449-c028139a-490c-4199-b72a-a28d94446ce3.png">